apps/condo/domains/meter/schema/RegisterMetersReadingsService.test.js

```diff
@@ -975,6 +975,69 @@ describe('RegisterMetersReadingsService', () => {
         expect(data2[0].id).toBe(metersReadings[0].id)
     })
 
+    test('prevent to create readings duplicates in one input', async () => {
+        const [o10n] = await createTestOrganization(adminClient)
+        const [property] = await createTestPropertyWithMap(adminClient, o10n)
+        const readingData = createTestReadingData(property)
+        const duplicateReadings = [readingData, readingData]
+
+        const [data] = await registerMetersReadingsByTestClient(adminClient, o10n, duplicateReadings)
+
+        expect(data).toEqual(expect.arrayContaining([expect.objectContaining({
+            id: expect.stringMatching(UUID_REGEXP),
+            meter: expect.objectContaining({
+                id: expect.stringMatching(UUID_REGEXP),
+                property: expect.objectContaining({
+                    id: property.id,
+                    address: property.address,
+                    addressKey: property.addressKey,
+                }),
+                unitType: duplicateReadings[0].addressInfo.unitType,
+                unitName: duplicateReadings[0].addressInfo.unitName,
+                accountNumber: duplicateReadings[0].accountNumber,
+                number: duplicateReadings[0].meterNumber,
+            }),
+        })]))
```

Comment:
Do you expect the data length to be 1 or not?
This code will not check the length, you should do it explicitly

Reply:
It should be 2 same rows in the data